language-antlr-utils/src/main/java/de/jplag/antlr/AbstractAntlrParserAdapter.java

```diff
@@ -69,7 +68,7 @@ private void parseFile(File file, TokenCollector collector) throws ParsingExcept
             for (ParseTree child : entryContext.children) {
                 treeWalker.walk(listener, child);
             }
-        } catch (IOException exception) {
+        } catch (Throwable exception) { // catching throwable to capture any exceptions thrown by ANTLR.
```

Comment:
Isn't Exception enough ?

Reply:
I also had this kind of Errors in mind. Nevertheless, I won't catch this kind of Error since this kind of errors can't be handled in a good way. Catching a OutOfMemoryError won't solve the problem as far as I remember :)

But I don't have a strong opinion on what to catch here :)

Reply:
One is better for API users, the other for application users. Let us go with `Exception` for now, we can always change it.